src/runtime_src/core/tools/xbmgmt2/SubCmdAdvanced.cpp

```diff
@@ -24,6 +24,7 @@
 #include "common/device.h"
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.